themes/bootstrap3/js/combined-search.js

```diff
@@ -0,0 +1,22 @@
+/*global VuFind, setupOpenUrlLinks, checkSaveStatuses, setupSaveRecordLinks */
+VuFind.combinedSearch = (function() {
```

Comment:
Works for me. Aside from my nitpick about the (VuFind) parameter, anything we need to worry about before merging?